### ISSUE TITLE
still maintained? req.once is not a function

### ISSUE THREAD

**Issue by cekvenich** [benhutchins/restifier.js#1]
Is this project still maintained?

**Comment by cekvenich** [comment]
restifier.js:344 Uncaught TypeError: req.once is not a function
    at _parse (restifier.js:344)

**Comment by benhutchins** [comment]
No, this is not maintained at the moment. If you opt to use it and make any improvements I will gladly merge them in.
